src/librustc_codegen_ssa/base.rs

```diff
@@ -835,9 +839,6 @@ impl CrateInfo {
             if tcx.is_profiler_runtime(cnum) {
                 info.profiler_runtime = Some(cnum.into());
             }
-            if tcx.is_sanitizer_runtime(cnum) {
-                info.sanitizer_runtime = Some(cnum.into());
-            }
```

Comment:
Why was this removed?

Reply:
Sorry about the confusion, these three lines was incorrectly kept due to rebase error. They were deleted in PR #65241.

Reply:
I've rebased this PR again, so these lines no longer shows up as "deleted".